Github-Users-List.md

```diff
@@ -0,0 +1,72 @@
+# Mobile Test - GitHub Users List App
+
+Make sure you read **all** of this document carefully, and follow the guidelines in it.
```

Comment:
"Make sure you read the whole document carefully and ..." sounds better.

Reply:
updating